sys/include/net/gnrc/netif/flags.h

```diff
@@ -129,6 +129,11 @@ enum {
  * @brief   Network interface is configured in raw mode
  */
 #define GNRC_NETIF_FLAGS_RAWMODE                   (0x00010000U)
+
+/**
+ * @brief   A Link Check request was scheduled
+ */
+#define GNRC_NETIF_FLAGS_LINK_CHECK                (0x00020000U)
```

Comment:
As someone who never worked with LoRa I ask myself immediately: What is a link check? Is this flag reusable for other network interface types?

If it is just LoRa-specific, why not introduce a separate flag field in `gnrc_netif_lorawan_t` and put the flag definition to `sys/include/net/gnrc/netif/lorawan.h`?

Reply:
Yes, I think is LoRaWAN specific. I will move them to the `lorawan.h` header

Reply:
addressed in https://github.com/RIOT-OS/RIOT/pull/11022/commits/067c753148176139cb420fa87a4361714ce9ac21. I added a commit to revert the changes to the `flags.h` header in https://github.com/RIOT-OS/RIOT/pull/11022/commits/c2052d0a2f695604889131a20fccf4625e09f2d4